### PR TITLE
chore: adding weth to common bases

### PR DESCRIPTION
**Pull request by snreynolds**

### CODE REVIEW

File: src/constants/routing.ts [hunk 1]
```diff
@@ -133,9 +135,15 @@ export const COMMON_BASES: ChainCurrencyList = {
     ExtendedEther.onChain(SupportedChainId.POLYGON),
     WRAPPED_NATIVE_CURRENCY[SupportedChainId.POLYGON],
   ],
+  [SupportedChainId.POLYGON]: [
+    ExtendedEther.onChain(SupportedChainId.POLYGON),
```
Comment (justindomingue) [inline src/constants/routing.ts:139]: nit. also defined at 135, small perf improvement by defining once.. not a big dal

File: src/constants/routing.ts [hunk 2]
```diff
@@ -133,9 +135,15 @@ export const COMMON_BASES: ChainCurrencyList = {
...
+    WRAPPED_NATIVE_CURRENCY[SupportedChainId.POLYGON],
+    WETH_POLYGON,
```
Comment (justindomingue) [inline src/constants/routing.ts:141]: should we add usdc here as well?

Comment (justindomingue) [reply src/constants/routing.ts:141]: and dai too maybe

**Review by justindomingue — APPROVED** [review]

File: src/constants/routing.ts (hunk 2 above)
Comment (willhennessy) [reply src/constants/routing.ts:141]: yes please, all the usual:  ethereum logo
ETH (n/a on polygon)
DAI
USDC
USDT
WBTC
WETH

**Pull request merged**